modules/ROOT/pages/7.2-release-notes.adoc

```diff
@@ -127,6 +127,17 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 
 Previously in {productname}, the `referrerpolicy` attribute was not considered as valid for iframe elements. In {productname} {release-version}, this attribute has been added to the schema, allowing users to include the `referrerpolicy` attribute in iframes.
 
+=== Improved Search & Replace dialog accessibility by changing placeholders to labels.
+// #TINY-10871
+
+In previous versions of {productname}, the Search & Replace dialog inputs were initially implemented with placeholders.
+
+As a consequence, the dialog was less accessible because the placeholders are removed once the user starts typing. This posed difficulties for people with cognitive disabilities who rely on constant visual cues for context.
```

Comment:
```suggestion
As a consequence, the dialog was less accessible because the input placeholders were hidden once the user started typing. This posed difficulties for users who rely on consistent visual cues for context.
```
Technically the placeholder isn't removed, it just goes away when there is text in the input field and can return if the field is empty again. I would say this is applicable to more than just people with cognitive disabilities.